test/scripts/database.ts

```diff
@@ -1,17 +1,18 @@
-'use strict';
-
-const should = require('chai').should(); // eslint-disable-line
-const path = require('path');
-const Promise = require('bluebird');
-const sinon = require('sinon');
-const fs = Promise.promisifyAll(require('fs'));
+// @ts-nocheck
```

Comment:
I add `@ts-nocheck` to all test files.
This workaround is temporarily.  I will update the test code.